src/libcollections/btree/node.rs

```diff
@@ -1418,19 +1296,21 @@ impl<'a, K, V> Handle<NodeRef<marker::Mut<'a>, K, V, marker::Internal>, marker::
                               right_edges,
                               new_right_len + 1);
                     right.correct_childrens_parent_links(0, new_right_len + 1);
-                },
-                (ForceResult::Leaf(_), ForceResult::Leaf(_)) => { }
-                _ => { unreachable!(); }
+                }
+                (ForceResult::Leaf(_), ForceResult::Leaf(_)) => {}
+                _ => {
+                    unreachable!();
+                }
             }
         }
     }
 }
 
-unsafe fn move_kv<K, V>(
-    source: (*mut K, *mut V), source_offset: usize,
-    dest: (*mut K, *mut V), dest_offset: usize,
-    count: usize)
-{
+unsafe fn move_kv<K, V>(source: (*mut K, *mut V),
+                        source_offset: usize,
+                        dest: (*mut K, *mut V),
+                        dest_offset: usize,
+                        count: usize) {
```

Comment:
This definitely benefits from having the `source` and `source_offset` on the same line and `dest` and `dest_offset` on the same line.